cmd/multi.go

```diff
@@ -352,19 +352,24 @@ func copyReqs() *jqs.Requirements {
 	return req
 }
 
-func scheduleStaticCopy(outputRoot, unique, partialDirMerge string, partialDirClean bool, s *scheduler.Scheduler) {
+func scheduleStaticCopy(outputRoot, unique, partialDirMerge string, partialDirClean bool,
+	s *scheduler.Scheduler) string {
 	var remove string
 
 	if partialDirClean {
 		remove = "--delete"
 	}
 
+	thisUnique := scheduler.UniqueString()
```

Comment:
Would it follow how the rest is done and make the dependency tree more obvious when looking at `wr status` if this was `unique+".mergedbs"`?

Reply:
Good idea; done.